tests/test_transformations/test_batch_normalize.py

```diff
@@ -180,6 +180,38 @@ def test_batch_normalize_q75(self):
         # print(ground_truth)
         self.assertEqual(np.allclose(ground_truth, normalized_array), True)
 
+    def test_batch_normalize_median_single_marker(self):
+        # Marker values and annotations
+        df = pd.DataFrame({
+            'marker1': [1, 5, 2, 6],
+            'batch_annotation': ["batch1", "batch2", "batch1", "batch2"]
+        })
+
+        # Assuming ingest_cells and concatinate_regions work as expected
+        adata = ingest_cells(df, "^marker.*", annotation="batch_annotation")

```

Comment:
Thanks @fangliu117 , let's use "marker1" instead of "^marker.*" to be more concise.